README.md

```diff
@@ -1 +1,40 @@
-# Morse-code-message
\ No newline at end of file
+## Getting Started
+
+This repository includes files with plain SQL that can be used to recreate a database:
```

Comment:
Please remove this description as it does not belong to the project. README should always be professional and without any errors in order to avoid confusion for the end user.